src/Avalonia.X11/XI2Manager.cs

```diff
@@ -309,11 +304,9 @@ private void OnDeviceEvent(IXI2Client client, ParsedDeviceEvent ev)
                     {
                         var pixelPoint = new PixelPoint((int)ev.RootPosition.X, (int)ev.RootPosition.Y);
                         var screen = _platform.Screens.ScreenFromPoint(pixelPoint);
-                        var screenBoundsFromPoint = screen?.Bounds;
-                        Debug.Assert(screenBoundsFromPoint != null);
```

Comment:
I don't remember why is this assert here, but maybe keep it? Sounds useful for debugging the particular condition it's checking for

Reply:
The `if` just below perfectly handles not finding a valid screen from the point, that's why I removed the assert.